tutorials/btp-cockpit-setup/btp-cockpit-setup.md

```diff
@@ -1,78 +1,78 @@
-
-    For more information, check out the blog [Exploring BTP's New Free Tier Plans](https://blogs.sap.com/2021/07/01/exploring-btps-new-free-tier-plans) by [Riley Rainey](https://people.sap.com/riley.rainey). And check out more posts and announcements on the [SAP BTP topic page](https://community.sap.com/topics/business-technology-platform) -- there's a quick link there to post your questions, too.
-
-
-[DONE]
-[ACCORDION-END]
-
-
-[ACCORDION-BEGIN [Step 2: ](Which to choose?)]
-
-Some services are available to try out for free in both free-tier and a trial account, some are available free only in free trial, and some are only available with a paid license.
-
-- If a service is available in both free tier and free trial, which should you choose? The decision may come down to whether your focus is on personal learning (free trial) or a more formal company-sponsored project (free tier).  Both approaches are available to you, and most existing SAP enablement content and code samples will run on both, too.
-
-- For tutorials with services not yet on free tier, you can still use a free trial account.
-
-[DONE]
-[ACCORDION-END]
-
-[ACCORDION-BEGIN [Step 2: ](How to set up an account)]
-
-Choose one of the following steps:
-
-- **Free Trial:** If you do not have an account of any type yet and you want to try out services for a limited amount of time, follow the instructions on the tutorial [Get a Free Account on SAP BTP Trial](hcp-create-trial-account).
-
-- **Free Tier:** If you want to explore SAP BTP without time limits, go to [Get an Account on SAP BTP to Try Out Free Tier Service Plans](btp-free-tier-account).
-
-    If your company has an existing consumption-based BTP Global Account, you can explore free tier services there. In such accounts, free tier plans will appear alongside paid service plans in the service marketplace.
-
-    > You will want to consider isolating your tutorial work from normal development and production activity in that global account, and isolating yourself is fairly easy: create a new subaccount. Then add appropriate free service plan entitlements to that subaccount, enable the services, and execute the tutorials there.
-
-Depending on your company's administration policies, you might need to have an authorized BTP administrator perform those tasks for you. In any event, instructions for these tasks are described here:
-
-- [Manage Entitlements Using the Cockpit](btp-cockpit-entitlements)
-
-- [Create a Service Instance for SAP BTP](btp-cockpit-instances)
-
-[DONE]
-[ACCORDION-END]
-
-
-
-
-
----
+---
+title: Get an SAP BTP Account for Tutorials
+description: Set up an SAP Business Technology Platform account so you can do the tutorials involving SAP BTP services and applications (trial or free tier).
+time: 5
+tags: [ tutorial>beginner, products>sap-business-technology-platform, tutorial>type>free-tier]

```

Comment:
Please correct tags (products -> software-product)

Reply:
Hey @MichaelCzcz  - sorry, I do not own these tutorials I just adapted some steps in reference to the documentation. Can you share more details what's wrong with the tags?
Annika